test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/RazorPagesWithoutModel_DesignTime.codegen.cs

```diff
@@ -25,20 +25,14 @@ public class TestFiles_IntegrationTests_CodeGenerationIntegrationTest_RazorPages
         }
         #pragma warning restore 219
         private static System.Object __o = null;
-        private global::DivTagHelper __DivTagHelper = null;
```

Comment:
What happened here? Looks like the test expectations/what's in the test aren't aligned anymore.